packages/firestore/lib/index.d.ts

```diff
@@ -1426,6 +1426,20 @@ export namespace FirebaseFirestoreTypes {
      * If set to false or omitted, the SDK throws an exception when it encounters properties of type undefined.
      */
     ignoreUndefinedProperties?: boolean;
+
+    /**
+     * 
+     * If set, controls the return value for server timestamps that have not yet been set to their final value.
+     * 
+     * By specifying 'estimate', pending server timestamps return an estimate based on the local clock. 
+     * This estimate will differ from the final value and cause these values to change once the server result becomes available.
+     * 
+     * By specifying 'previous', pending timestamps will be ignored and return their previous value instead.
+     * 
+     * If omitted or set to 'none', null will be returned by default until the server value becomes available.
+     * 
```

Comment:
can you add a bold disclaimer here that is is currently android only? Do you have any plans to plumb it through for iOS out of curiosity?

Reply:
Can do.
I might take a stab at iOS in the next couple of days, but it's been a long time since I did anything in Objective C

Reply:
iOS looks ace!